pkg/planner/core/rule/util/BUILD.bazel

```diff
@@ -5,11 +5,13 @@ go_library(
     srcs = [
         "build_key_info_misc.go",
         "misc.go",
+        "rule_constant_propagation.go",
```

Comment:
you can rename the file as `constant_propagation_misc.go` and then `make bazel_prepare`